setup.py

```diff
@@ -36,6 +36,7 @@ def run(self):
         _build_py.run(self)
 
 
+render = ["dvc_render"]
```

Comment:
Should this be optional? It seems like this is the basic functionality of dvclive.